### DISCUSSION TITLE
Offline Error with iboss

### DISCUSSION THREAD

**Discussion by Isaac74429** [3kh0/ext-remover#31] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i did the steps and disabled iboss but i cannot veiw the websites it says that im offline

**Comment by 3kh0** [comment, marked as answer]
can you send any screenshots?

**Reply by Isaac74429** [reply]
![Screenshot 2022-10-05 10 43 40 AM](https://user-images.githubusercontent.com/114715036/194103588-301f3aef-4ec7-4f67-ab52-0f68613c7461.png)

Thsi is what happens

**Reply by 3kh0** [reply]
are you going to the chrome webstore?

**Reply by Isaac74429** [reply]
webstorex?

**Reply by Isaac74429** [reply]
Too make sure what im doing is correct you have to open the link to chromestorex and then use the ingot bookmark correct?

**Reply by 3kh0** [reply]
yes

**Reply by Isaac74429** [reply]
Does it work with iboss though?

**Reply by 3kh0** [reply]
yes

**Reply by Gumkid3** [reply]
This is because you are using school wifi (or wifi that the school controls) most schools put second-hand blockers on their wifi's, so use a wifi that isn't from school
